### PR TITLE
CR-1105659 use device bdf in python tests command line args instead of device index

### PR DESCRIPTION
**Pull request by rajkumar-xilinx**
#### Problem solved by the commit
Python applications (as part of xbutil validate) are using device indexes in command line args.
 This is causing a problem when running xbutil tests run on multi card setup (ex: Microsoft Azure) concurrently.
Example:
   1. Issue reset on device0 (has index 0), and issue validate on device1 (has index 1).
   2. In device0 reset sequence, it resets the card and removes the index from devicesList.
      Hence, device1 index become 0.
   3. This same issue not seen if validate test run on device0 and reset on device1.
      Since, reset of device1 removes the index from deviceList, and doesn't affect device0 index.

Signed-off-by: Durga prasad Kotipalli <dkotipal@xilinx.com>
Signed-off-by: Rajkumar Rampelli <rajkumar@xilinx.com>

Validate tests failure is solved.

#### Bug / issue (if any) fixed, which PR introduced the bug, how it was discovered
CR-1105659

#### How problem was solved, alternative solutions (if any) and why they were rejected
Added change in XRT tests.

#### Risks (if any) associated the changes in the commit
NA

#### What has been tested and how, request additional testing if necessary
xbutil reset and validate tests on multi card setup.

#### Documentation impact (if any)
NA

### CODE REVIEW

**Review by AShivangi — APPROVED** [review]

**Comment by rajkumar-xilinx** [comment]
retest this please

**Review by chvamshi-xilinx — APPROVED** [review]

**Comment by chvamshi-xilinx** [comment]
Need to back port this to 2022.1

**Comment by xilinxgitops** [comment]
retest this please

**Comment by salindac** [comment]
retest this please

**Comment by chvamshi-xilinx** [comment]
retest this please

**Comment by rajkumar-xilinx** [comment]
Hi @rozumx & @arpitxilinx  Please review this change, thank you

File: src/python/xrt_binding.py [hunk 1]
```diff
@@ -1008,7 +1008,7 @@ def xrtDeviceOpenByBDF(bdf):
     """
```
Comment (rozumx) [inline src/python/xrt_binding.py:1008]: Please update the copyright date - Line 2

File: tests/python/utils_binding.py [hunk 2]
```diff
@@ -33,7 +33,7 @@ def __init__(self):
         self.halLogFile = None
```
Comment (rozumx) [inline tests/python/utils_binding.py:33]: Please update the copyright date - Line 2

**Review by rozumx — APPROVED** [review]

**Review by arpitxilinx — APPROVED** [review]

**Comment by chvamshi-xilinx** [comment]
Please update the copyright in next PR. Merging this as it is needed for customer.

**Pull request merged**